be/src/io/cache/cached_remote_file_reader.cpp

```diff
@@ -312,15 +312,20 @@ Status CachedRemoteFileReader::read_at_impl(size_t offset, Slice result, size_t*
 }
 
 void CachedRemoteFileReader::_update_state(const ReadStatistics& read_stats,
-                                           FileCacheStatistics* statis) const {
+                                           FileCacheStatistics* statis,
+                                           bool is_inverted_index) const {
     if (statis == nullptr) {
         return;
     }
     if (read_stats.hit_cache) {
         statis->num_local_io_total++;
         statis->bytes_read_from_local += read_stats.bytes_read;
     } else {
-        statis->num_remote_io_total++;
+        if (is_inverted_index) {
+            statis->num_inverted_index_remote_io_total++;
```

Comment:
You can inc both num_remote_io_total and num_inverted_index_remote_io_total.

Reply:
done